chia/_tests/core/mempool/test_mempool.py

```diff
@@ -2932,45 +2931,8 @@ def test_items_by_feerate(items: list[MempoolItem], expected: list[Coin]) -> Non
         last_fpc = mi.fee_per_cost
 
 
-# make sure that after failing to pick 3 fast-forward spends, we skip
-# FF spends. create_block_generator2() does not stop trying FF spends after 3
-# failures, it keeps going. The new function (create_block_generator2()) does
-# not have a limit on FF and dedup spends so it tries all 5. make_test_coins()
-# only returns 5 coins
-@pytest.mark.anyio
-@pytest.mark.parametrize("old, expected", [(True, 3), (False, 5)])
-async def test_skip_error_items(old: bool, expected: int) -> None:
```

Comment:
why did you remove this test? As far as I can tell, this limit is still in effect in `create_bundle_from_mempool()`